apps/api/.env.schema

```diff
@@ -1,9 +1,24 @@
-# Server
-PORT=3000
-IS_PROD=false
+# -- Server --
+# PORT=3000
+SESSION_KEYS=newsecret,oldsecret
+# SESSION_MAX_AGE=1800000
```

Comment:
Why are some of these commented out?

Reply:
To show that the env variable is available if you want to change it, but doesn't need to be set by default. The value shown is the default value.